File: LPPAWeb/PatenteFamilia/C#/SL/Objects/exceptionlogging.cs
using System;
using System.Collections.Generic;
using System.Text;
using Microsoft.Practices.EnterpriseLibrary.ExceptionHandling;
using Microsoft.Practices.EnterpriseLibrary.Logging;
//using Microsoft.Practices.EnterpriseLibrary.Data;

namespace SL
{
    public static class ExceptionLogging
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="message"></param>
        public static void LogWriter(string message)
        {
            LogEntry logentry = new LogEntry();
            logentry.Message = message;
            logentry.Title = "Program";
            logentry.TimeStamp = DateTime.Now;
            Logger.Write(logentry);
        }
        /// <summary>
        /// 
        /// </summary>
        /// <param name="ex"></param>
        public static void HandleException(Exception ex)
        {
            ExceptionLogging.LogWriter(ex.Message);
            ExceptionPolicy.HandleException(ex, "Exception Policy");
        }
    }
}

File: LPPAWeb/PatenteFamilia/VB.NET/SL/ExceptionLogging.vb
Imports Microsoft.Practices.EnterpriseLibrary.Logging
Imports Microsoft.Practices.EnterpriseLibrary.ExceptionHandling

Public Class ExceptionLogging

    Public Shared Sub HandleException(ByVal ex As Exception)
        'ExceptionLogging.LogWriter(ex.Message)
        'ExceptionPolicy.HandleException(ex, "Exception Policy")
    End Sub


    Public Shared Sub LogWriter(ByVal message As String)
        'Dim logentry As New LogEntry With { _
        '                                   .Message = message, _
        '                                   .Title = "Program", _
        '                                   .TimeStamp = DateTime.Now _
        '                               }
        'Logger.Write(logentry)
    End Sub

End Class
